pkg/utils/utils.go

```diff
@@ -51,6 +56,23 @@ func init() {
 	ClusterType = os.Getenv("CLUSTER_TYPE")
 }
 
+// IsKernelCmdLineParamSet This checks if the kernel cmd line is set properly.
+func IsKernelCmdLineParamSet(param string, chroot bool) (bool, error) {
```

Comment:
+1 great idea. imo better name is then: `getCurrentKernelArgs`. I prefer "args" instead of "params" for sure (the difference is quite formal).  In addition, since we are reading /proc/cmdline we are getting the currently running cmdline args so I prefer to add `Current` in the name of the function. That would allow us to also create a similar function that gets the kernel args for the next boot.

Btw, also rename the PR to "Kernel args set by.."

Reply:
Yes I made this change to GetCurrentKernelArgs(chroot bool) (string, error) . I am keeping it as a string because the "cmdLine" could have the same argument repeated.